metabase/metabase
# Add 12-hour clock support to events

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Adds 12-hour clock support to `TimeInput`
- Adds 12-hour clock support to `DateSelector`

How to test:
- Go to Admin > Settings > Localization and set 12-hour clock
- Go to a collection and add/edit an event. Make sure it's possible to set time in am/pm format
- Go to Admin > Settings > Localization and set 24-hour clock
- Go to a collection and add/edit an event. Make sure the time is formatted with 24-hour clock

<img width="658" alt="Screenshot 2022-02-23 at 21 12 01" src="https://user-images.githubusercontent.com/8542534/155381401-5c684302-bc3e-44e1-8adb-c13afeef1d3f.png">
<img width="663" alt="Screenshot 2022-02-23 at 21 12 10" src="https://user-images.githubusercontent.com/8542534/155381417-83209d7c-1913-411c-a670-7bac81850dcf.png">
<img width="679" alt="Screenshot 2022-02-23 at 21 12 23" src="https://user-images.githubusercontent.com/8542534/155381421-3e9dac22-e294-48c5-9b75-66689d344947.png">
<img width="660" alt="Screenshot 2022-02-23 at 21 12 30" src="https://user-images.githubusercontent.com/8542534/155381426-2992e6f4-66cb-4c39-9dcb-5d10eb18a205.png">
